root/elections/Show.js

```diff
@@ -0,0 +1,39 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const React = require('react');
+const Layout = require('../layout');
+const {l} = require('../static/scripts/common/i18n');
+const ElectionDetails = require('./ElectionDetails');
+const ElectionVotes = require('./ElectionVotes');
+const ElectionVoting = require('./ElectionVoting');
+
+const Show = () => {
+  const election = $c.stash.election;
```

Comment:
Same comment as I left on root/elections/Index.js about sending this as a prop instead (but don't worry about `$c.user`).